libs/components/forms/src/lib/modules/input-box/input-box.component.scss

```diff
@@ -394,9 +420,32 @@ sky-input-box {
       }
 
       textarea.sky-form-control {
-        margin-top: 0;
+        margin-top: -23px;
         padding-top: 0;
         resize: vertical;
+        border-top: 26px solid transparent;
```

Comment:
Using a `border-top` here is a little unconventional - but allows the autofill styles to fill the entire box while not allowing the text to overlap with the label - which is what happens if we use standard padding.